src/script/script_gui.cpp

```diff
@@ -120,11 +122,14 @@ struct ScriptListWindow : public Window {
 					DrawString(tr, this->slot == OWNER_DEITY ? STR_AI_CONFIG_NONE : STR_AI_CONFIG_RANDOM_AI, this->selected == -1 ? TC_WHITE : TC_ORANGE);
 					tr.top += this->line_height;
 				}
+				StringID str = this->show_all ? STR_AI_CONFIG_NAME_VERSION : STR_JUST_RAW_STRING;
 				int i = 0;
 				for (const auto &item : *this->info_list) {
 					i++;
 					if (this->vscroll->IsVisible(i)) {
-						DrawString(tr, item.second->GetName(), (this->selected == i - 1) ? TC_WHITE : TC_ORANGE);
+						SetDParamStr(0, item.second->GetName());
+						SetDParam(1, item.second->GetVersion());
```

Comment:
Honest question: Is it not a problem or bad form to set a DParam that never gets consumed?

Reply:
String parameters are in a fixed size array of uint64 values, so consumed or not doesn't matter.

Reply:
Thanks, good to know. 😃